ICRCs/ICRC-97/ICRC-97.md

```diff
@@ -0,0 +1,101 @@
+
+Below list of purpose values are part of the ICRC-97 standard, this list could be extended by other standards.
+Purpose values can define additional asset properties.
+
+| Purpose Value  | Description                                                                                                                                                 |
+|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| icrc97:image   | Original size image that is shown on e.g. item details page. Additional optional `width` and `height` properties define the dimensions in number of pixels. |
+| icrc97:preview | Small image meant as preview within e.g. a list of items. Additional optional `width` and `height` properties define the dimensions in number of pixels.    |
+
+The following table lists all (optional) attribute properties.
+
+| Attribute Property | Optional | ICRC-3 Type                | JSON Type        | Description                                 |
+|--------------------|----------|----------------------------|------------------|---------------------------------------------|
+| value              | No       | variant { Text; Nat; Int } | string \| number | Value of the trait.                         |
+| trait_type         | No       | variant { Text }           | string           | Name of the trait.                          |
+| display_type       | Yes      | variant { Text }           | string           | Indicate how attribute should be displayed. |
+
+Below list of display types are part of the ICRC-97 standard, this list could be extended by other standards.
+Display types can define additional attribute properties.
+
+| Display Type Value      | Description                                                                                                                                                   |
+|-------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| icrc97:property         | Shows attribute as property with e.g. rarity, this is the default display type.                                                                               |
+| icrc97:date             | Shows attribute as date, expects epoch timestamp number in milliseconds.                                                                                      |
+| icrc97:time             | Shows attribute as date with time, expects epoch timestamp number in milliseconds.                                                                            |
+| icrc97:rank             | Shows attribute as progress rank e.g. 4 of 10, expects number and additional `max_value` property with a number.                                              |
```

Comment:
can you provide an example on how 4 of 10 would be defined?

Reply:
I'll add examples to the spec